package.json

```diff
@@ -33,8 +33,8 @@
     "docs:watch": "npm run docs:prepare && gitbook serve",
     "docs:publish": "npm run docs:clean && npm run docs:build && ./scripts/publish-docs",
     "flow": "eslint packages",
-    "todo": "leasot 'packages/**/*.js' --ignore '**/node_modules','**/dist','**/lib'",
-    "todo-staged": "leasot -x --reporter markdown 'packages/**/*.js' --ignore '**/node_modules','**/dist','**/lib' > TODO.md && git add TODO.md",
+    "todo": "leasot 'packages/*/src/**.js'",
```

Comment:
Different thread: what's the difference between `todo` and `todo-staged` and why doesn't one use the other?

Reply:
Its weird.

`todo` is meant to just be run in the terminal to quickly print out all the TODO's.
`leasot 'packages/*/src/**.js' -x`

If we extend that command in todo-staged so it becomes

`npm run todo -- --reporter markdown > TODO.md`

it actually includes all below in the piping to file:

> @ todo /Users/*****/*****/nodejs
> leasot 'packages/*/src/**.js' -x "--reporter" "markdown"

When running it by itself, without extending original `todo`, I dont see this problem.